cfme/rest/gen_data.py

```diff
@@ -238,7 +240,7 @@ def _finished():
 
 
 def a_provider():
```

Comment:
Wondering if we should normalize all these
we have ```a_provider``` here and ```setup_a_provider``` else where. I guess we should look at what the fixture actually does. ```a_provider``` is possibly better than ```setup_a_provider```, thoughts?